repository.rosinstall

```diff
@@ -12,3 +12,8 @@
     uri: https://github.com/shadow-robot/ros_ethercat.git
     version: 'kinetic-devel'
     local-name: ros_ethercat
+
+- git:
+    uri: https://github.com/shadow-robot/sr_utils.git
+    version: 'F_adding_libraries_from_shadow_r_ethercat'
```

Comment:
Not sure how we do this normally, but this will have to be `kinetic-devel` after the other one is merged.

Reply:
@toliver yes, I'm keeping tabs on that